File: 0344-reverse-string/0344-reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s);
};
/*********************************************************************/
void Solution::reverseString(vector<char>& s) {
    int size = s.size(), left, right;
    char aux;
    for (left = 0, right = size-1; left < right; ++left, --right) {
        aux = s[left];
        s[left] = s[right];
        s[right] = aux;
    }
    
}
/*********************************************************************/

File: 0344-reverse-string/0344-reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        sLen = len(s)
        for i in range(sLen//2):
            #swap is awesome in python
            s[i], s[sLen-1-i] = s[sLen-1-i], s[i]